### ISSUE TITLE
Running timeline along while receiving data from Studio

### ISSUE THREAD

**Issue by FrederikLarsenRokoko** [Rokoko/rokoko-studio-live-blender#64]
Hi guys, Frederik from technical support here,
We have a customer that would like to be able to run the timeline in Blender along while he is receiving mocap data from Studio.
He needs it for something about using Blender to relay OSC data for sound control externally. And he can't do that without the timeline in Blender running.
Seems like it could also be a good feature for others to be able to have things in the scene moving, while simultaneously being able to receive mocap data.
So, in short, would it be possible to create a way to start receiving mocap data from Studio without pausing the timeline playback at the same time?
Thanks
